docker/Dockerfile

```diff
@@ -51,20 +62,21 @@ RUN ln -s /user-data /home/$DOCKER_USER/data
 #RUN curl -LO https://raw.githubusercontent.com/broadinstitute/viral-ngs-deploy/master/easy-deploy-script/easy-deploy-viral-ngs.sh
 USER root
 ADD ./easy-deploy-script/easy-deploy-viral-ngs.sh /home/$DOCKER_USER/easy-deploy-viral-ngs.sh
-RUN chmod +x ./easy-deploy-viral-ngs.sh
-RUN chown $DOCKER_USER:$DOCKER_USER ./easy-deploy-viral-ngs.sh
+RUN chmod +x ./easy-deploy-viral-ngs.sh && \
+    chown $DOCKER_USER:$DOCKER_USER ./easy-deploy-viral-ngs.sh
 USER $DOCKER_USER
 ENV VIRAL_NGS_VERSION $VIRAL_NGS_VERSION
+
 RUN echo "Running easy install with viral-ngs version $VIRAL_NGS_VERSION"
-RUN bash -c 'echo $(if [ "$VIRAL_NGS_VERSION" == "" ]; then echo "./easy-deploy-viral-ngs.sh setup"; else echo "./easy-deploy-viral-ngs.sh setup --viral-ngs-version $VIRAL_NGS_VERSION" ; fi)'
```

Comment:
Yup! As things are currently [configured](https://github.com/broadinstitute/viral-ngs/blob/master/DEVELOPMENT_NOTES.md) (as of last week), the main viral-ngs repository has a [deploy hook](https://github.com/broadinstitute/viral-ngs/blob/master/.travis.yml#L64) that [renders, builds, and uploads](https://github.com/broadinstitute/viral-ngs/blob/master/travis/deploy.sh#L44) a conda package for viral-ngs when a new [release](https://github.com/broadinstitute/viral-ngs/releases) is tagged. If the conda build is successful, a script[ kicks off](https://github.com/broadinstitute/viral-ngs/blob/master/travis/trigger-tests-in-other-repo.sh#L16) Travis builds in viral-ngs-deploy to build a docker image automatically and push it to [Docker Hub](https://hub.docker.com/r/broadinstitute/viral-ngs/tags/).